### PR TITLE
Formatting test duration in status page to hh:mm:ss format for better readability

### PR DESCRIPTION
**Pull request by vijayamanohararaj**
The current test status page for UI and eyes test shows the duration in seconds, which can take a bit for someone to parse and infer how long the test would run. For better readability, formatting the duration to saying <hh> hrs <mm> mins <ss> seconds.

Additionally fixed lint errors that were blocking commit on this file.

## Links
None - following up as this was suggested as an idea during DOTD shadow

## Testing story

Below are the screen shots for tests that are less than a min, around a min, min and a few seconds and an hour long

16 seconds
image

75 seconds
image

180 seconds
image

3600 seconds
image

7600 seconds
image

## Deployment strategy
Will be rolled out with regular DOTD deployments, and automated test/staging deployments

## PR Checklist:

- [x] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [x] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [x] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: dashboard/test/ui/test_status.js [hunk 1]
```diff
@@ -125,9 +125,14 @@ Test.prototype.updateView = function() {
   // Update row appearance
   row.className = this.status;
   if (succeeded || failed) {
+    var formattedDuration = new Date(this.duration * 1000)
+      .toISOString() // Sample formatting - 1970-01-01T00:00:42.000Z
+      .slice(11, 19) // extracts the time from the above string
+      .split(":");
+
```
Comment (pablo-code-org) [inline dashboard/test/ui/test_status.js]: I would just create a method called getFormattedDate() that returns the string 
${formattedDuration[0]} hr ${formattedDuration[1]} min ${formattedDuration[2]}

The reason is that it is the only thing this variable is doing and will keep the logic in a single place. We don't really use the rest of the values in the vector for anything else

Comment (Hamms) [reply dashboard/test/ui/test_status.js]: I would also recommend you avoid manually slicing and splitting the ISO string to extract the data you want, and instead use `formattedDuration.getUTCHours()`, `formattedDuration.getUTCMinutes()`, and
 `formattedDuration.getUTCSeconds()`.

Comment (vijayamanohararaj) [reply dashboard/test/ui/test_status.js]: @pablo-code-org , I'm assuming with the simplification Elijah suggested, we don't have the vector and only a date object. So, leaving the code inline.

Thanks  @Hamms  on the pointer on getUTCHours, this is so much cleaner. I had originally wanted to do this and was using getHours instead, which gave the wrong value.

**Review by pablo-code-org — APPROVED** [review]
looks good to me :)

File: dashboard/test/ui/test_status.js (hunk 1 above)
Comment (Hamms) [reply dashboard/test/ui/test_status.js]: It really is silly that the default/standard methods aren't in UTC by default with something like a `getTimeZoneHours` alternative for the subjective case

**Review by Hamms — APPROVED** [review]

**Pull request merged**